zio-kafka-test-utils/src/main/scala/zio/kafka/embedded/Kafka.scala

```diff
@@ -55,5 +60,34 @@ object Kafka {
     ZIO.acquireRelease(ZIO.attempt(Kafka.Sasl(EmbeddedKafkaService(EmbeddedKafka.start()))))(_.value.stop())
   }
 
+  val sslEmbedded: ZLayer[Any, Throwable, Kafka] = ZLayer.scoped {
+    val listener = s"${SecurityProtocol.SSL}://localhost:$defaultKafkaPort"

```

Comment:
```suggestion
    val listener = s"${SecurityProtocol.SSL.name}://localhost:$defaultKafkaPort"

```

Reply:
BTW, why not just hardcoding "SSL://" here? It seems that you're hardcoding it there anyway: https://github.com/conduktor/zio-kafka/pull/136/files#diff-34882ab1b3cd302099574477c6edcb9d14fec7ce81e9b62d5816b6cfb315774aR84 🤔

Reply:
Fixed